### ISSUE TITLE
Auditbeat 7.5 

### ISSUE THREAD

**Issue by jdelvecchio** [apolloclark/ansible-role-auditbeat#3]
Hey, thanks for the role, any plans to maintain it with the most recent version (7.5 at the moment) ?

Regards,
Julien

**Comment by apolloclark** [comment]
No, sorry, busy with work. Feel free to open a Pull Request to update it.
